tests/c_resources_test.go

```diff
@@ -17,9 +18,14 @@ func Test_cresources(t *testing.T) {
 	}
 	defer db.DropDB(os.Getenv("DB_NAME"))
 
-	// create user
-	user := db.User{Username: "foo", Password: "bar"}
-	tokenStr, createdUser := createUserAndLogin(t, app, user)
+	// create user with admin rights
+	user := db.User{Username: "foo", Password: "bar", Type: db.AdminUT}
```

Comment:
Adding an initial setup-step that does exactly this would also be a good idea. You would have to make it mandatory for the superAdmin to change the default credentials right after though.

Reply:
Yes, exactly that. Would be fine if not included in this PR but Ill open an issue on this